physionet-django/console/templates/console/event_agreement_list.html

```diff
@@ -0,0 +1,93 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block local_css %}
+  <link rel="stylesheet" type="text/css" href="{% static 'custom/css/pagination.css' %}">
+{% endblock %}
+
+{% block title %}{{ SITE_NAME }} - Event Agreements{% endblock %}
+
+{% block content %}
+ <div class="card mb-3">
+  <div class="card-header">
+    Event Agreement <span class="badge badge-pill badge-info">{{ event_agreements|length }}</span>
+  </div>
+  <div class="card-body">
+    {% if event_agreements %}
+    <div class="table-responsive">
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Version</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+        {% for event_agreement in event_agreements %}
+            <tr>
+                <td>
+                  {{ event_agreement.name }}
+                  {% if event_agreement.is_active %}
+                    <span class="badge badge-success">Active</span>
+                  {% else %}
+                    <span class="badge badge-danger">Inactive</span>
+                  {% endif %}
+                </td>
+                <td>{{ event_agreement.version }}</td>
+                <td>
+                  <a href="{% url 'event_agreement_detail' event_agreement.pk %}" class="btn btn-sm btn-primary" role="button">Edit</a>
+                  <a href="{% url 'event_agreement_new_version' event_agreement.pk %}" class="btn btn-sm btn-success" role="button">New version</a>
+                  <button type="button" class="btn btn-sm btn-danger " data-toggle="modal" data-target="#remove-event-agreement-{{ event_agreement.pk }}">Delete</button>
+
+                  <div class="modal fade" id="remove-event-agreement-{{ event_agreement.pk }}" tabindex="1" role="dialog" aria-hidden="true">
+                    <form method="post" action="{% url 'event_agreement_delete' event_agreement.pk %}">
+                      {% csrf_token %}
+                      <div class="modal-dialog" role="document">
+                        <div class="modal-content">
+                          <div class="modal-header">
+                            <h5 class="modal-title">Delete DUA</h5>
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                              <span aria-hidden="true">&times;</span>
+                            </button>
+                          </div>
+                          <div class="modal-body">
+                              <p>Are you sure you want to delete the DUA?</p>
```

Comment:
```suggestion
                              <p>Are you sure you want to delete the agreement?</p>
```

Reply:
oh man. Thats an embarrassing mistake. i will remember to review my own PR first going forward.